README.md

```diff
@@ -1,6 +1,6 @@
 # Tobira: an Opencast Video Portal
 
-![CI Status (master)](https://img.shields.io/github/workflow/status/elan-ev/tobira/CI/master)
+![CI Status (master)](https://github.com/elan-ev/tobira/workflows/CI/badge.svg)
```

Comment:
Does this also show the badge for `master`? Or is that the latest run, which could be any failing PR?

Reply:
Not necessarily `master` but whatever is the default branch. For Tobira, this is currently `master`